BubbleSorting/SortingAlgorithm.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BubbleSorting
+{
+    class SortingAlgorithm
+    {
+        /// <summary>
+        /// Sum of element numbers from a row of array
+        /// </summary>
+        /// <param name="arr">An array from SortByRowSum(int[][] arr)</param>
+        /// <returns>Sum of element numbers from a row of array</returns>
+        public int SumUpNumbers(int[] arr)  
```

Comment:
Why is this method public? You've never used it outside the class?
You should make private methods If you going to use them only inside the class.